.github/workflows/ci-linux.yml

```diff
@@ -0,0 +1,139 @@
+  # push:
+
+  # Manual trigger using the Actions page. May remove when integration complete.
+  workflow_dispatch:
+
+jobs:
+  x86:
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Create Build Environment
+      run:
+        sudo apt-get -y install doxygen vera++ cmake zlib1g-dev libsnappy-dev;
+        sudo apt-get -y install g++-multilib;
+
+    - name: Run Suite
+      working-directory: ${{ github.workspace }}
+      shell: bash
+      run: ./suite/runsuite_wrapper.pl travis $EXTRA_ARGS
+      env:
+        DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY: no
+        DEPLOY: no
+        EXTRA_ARGS: "32_only"
+
+  x86-64:
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Create Build Environment
+      run:
+        sudo apt-get -y install doxygen vera++ cmake zlib1g-dev libsnappy-dev;
+        sudo apt-get -y install g++-multilib;
+
+    - name: Run Suite
+      working-directory: ${{ github.workspace }}
+      shell: bash
+      run: ./suite/runsuite_wrapper.pl travis $EXTRA_ARGS
+      env:
+        DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY: no
+        DEPLOY: no
+        EXTRA_ARGS: "64_only"
+
+  archxx-cross-compile:
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Create Build Environment
+      run:
+        sudo apt-get -y install doxygen vera++ cmake zlib1g-dev libsnappy-dev;
+        sudo apt-get -y install g++-arm-linux-gnueabihf g++-aarch64-linux-gnu;
+
+    - name: Run Suite
+      working-directory: ${{ github.workspace }}
+      shell: bash
+      run: ./suite/runsuite_wrapper.pl travis $EXTRA_ARGS
+      env:
+        DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY: yes
+        DEPLOY: no
+
+  android-arm-cross-compile:
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Create Build Environment
+      # Manually force using ld.bfd, setting CMAKE_LINKER does not work.
+      run: |
+        sudo apt-get -y install doxygen vera++ cmake zlib1g-dev libsnappy-dev
+        cd /tmp
+        wget https://dl.google.com/android/repository/android-ndk-r10e-linux-x86_64.zip
+        unzip -q android-ndk-r10e-linux-x86_64.zip
+        android-ndk-r10e/build/tools/make-standalone-toolchain.sh --arch=arm \
+          --toolchain=arm-linux-androideabi-4.9 --platform=android-21 \
+          --install-dir=/tmp/android-gcc-arm-ndk-10e
+        ln -sf ld.bfd /tmp/android-gcc-arm-ndk-10e/arm-linux-androideabi/bin/ld
+        ln -sf arm-linux-androideabi-ld.bfd \
+          /tmp/android-gcc-arm-ndk-10e/bin/arm-linux-androideabi-ld
+
+    - name: Run Suite
+      working-directory: ${{ github.workspace }}
+      shell: bash
+      env:
+        DYNAMORIO_CROSS_ANDROID_ONLY: yes
+        DEPLOY: no
+        DYNAMORIO_ANDROID_TOOLCHAIN: '/tmp/android-gcc-arm-ndk-10e'
```

Comment:
nit: Why is this env value in quotes while the others aren't

Reply:
Fixed it.